src/material-experimental/mdc-input/input.spec.ts

```diff
@@ -1094,6 +1094,23 @@ describe('MatMdcInput with forms', () => {
     expect(formField._control.empty).toBe(false);
   }));
 
+  it('should update notch size after changing appearance to outline', fakeAsync(() => {
+    const fixture = createComponent(MatInputWithAppearance);
+    fixture.detectChanges();
+
+    expect(document.querySelector('.mdc-notched-outline__notch')).toBe(null);
+
+    fixture.componentInstance.appearance = 'outline';
+    fixture.detectChanges();
+
+    let notch = document.querySelector('.mdc-notched-outline__notch')! as HTMLElement;
+    expect(notch.style.width).toBe('');
+
+    document.querySelector('input')!.focus();
+    fixture.detectChanges();
+
+    expect(notch.style.width).not.toBe('');
```

Comment:
Maybe this should be `toBeTruthy`? Otherwise this would pass for `null` or `undefined`.